sdk/Lusid.Sdk.Tests/tutorials/MarketData/MarketDataRules.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.MarketData
+{
+    public class MarketDataRules : TutorialBase
+    {
+        [Test]
+        public void DemoMarketDataSpecificRules()
+        {
+            // set up scopes: one for recipes, one for holding quotes from a generic source, and one for holding specific quotes that will override ones from the generic scope
+            var testScope = "testScope";
+            var genericScope = "genericScope";
+            var specificScope = "specificScope";
+
+            // create an equity call option for TSLA to value
+            var testNow = new DateTimeOffset(2019, 01, 01, 0, 0, 0, TimeSpan.Zero);
+            var instrument = new EquityOption(testNow.AddMonths(-1), testNow.AddMonths(+1), testNow.AddMonths(+1), EquityOption.DeliveryTypeEnum.Cash,
+                EquityOption.OptionTypeEnum.Call, 90m, "USD", EquityOption.UnderlyingIdentifierEnum.RIC, "TSLA", LusidInstrument.InstrumentTypeEnum.EquityOption);
+
+            // upsert two quotes with different values: one is upserted to the generic scope, one is upserted to the specific scope
+            var genericQuote = TestDataUtilities.BuildQuoteRequest("TSLA", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 100m, "USD", testNow);
+            var genericQuoteResponse = _quotesApi.UpsertQuotes(genericScope, genericQuote);
+            ValidateQuoteUpsert(genericQuoteResponse, genericQuote.Count);
+            var specificQuote = TestDataUtilities.BuildQuoteRequest("TSLA", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 120m, "USD", testNow);
+            var specificQuoteResponse = _quotesApi.UpsertQuotes(specificScope, specificQuote);
+            ValidateQuoteUpsert(specificQuoteResponse, specificQuote.Count);
+
+            // instruct lusid to value equity options with ConstantTimeValueOfMoney (i.e. intrinsic value)
+            var modelRules = new VendorModelRule(VendorModelRule.SupplierEnum.Lusid, "ConstantTimeValueOfMoney", "EquityOption", "{}");
+            var pricingContext = new PricingContext(modelRules: new List<VendorModelRule> {modelRules});
+
+            // make two market data rules:
+            // the first is a generic rule that all RIC prices should be looked for in the generic scope
+            // the second is a specific rule that all RIC prices requested by USD-denominated EquityOption instruments should be looked for in the specific scope
+            var genericRule = new MarketDataKeyRule("Equity.RIC.*", "Lusid", genericScope, MarketDataKeyRule.QuoteTypeEnum.Price, "mid");
+            var specificRule = new MarketDataSpecificRule("Equity.RIC.*", "Lusid", specificScope, MarketDataSpecificRule.QuoteTypeEnum.Price, "mid",
```

Comment:
Can we rename this to `specificRuleForUsdEquityOptions`? And similarly make the variable name and recipe names more descriptive?